mappings/net/minecraft/screen/NamedScreenHandlerFactory.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_1723 net/minecraft/screen/NamedScreenHandlerFactory
+	METHOD method_6761 createMenu (Lnet/minecraft/class_1962;Lnet/minecraft/class_1963;)Lnet/minecraft/class_1983;
```

Comment:
createScreen ?

Reply:
upstream calls it `createMenu` even if it's not supposed to be called that